src/domains/charts/providers/menu/menu.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import styled from "styled-components"
+import { Flex } from "@netdata/netdata-ui"
+import { MenuGroupContainer } from "domains/charts/providers/menuGroup"
+import { ActiveMenuConsumer } from "domains/charts/providers/active"
+import { MenuConsumer } from "./context"
+
+export const Menu = ({ ids, activeMenuId, onMenuItemClick, onGroupItemClick, ...rest }) => (
+  <Flex as="ul" column {...rest}>
+    {ids.map(id => {
+      const active = activeMenuId === id
+      return (
+        <MenuGroupContainer
+          key={id}
+          id={id}
+          active={active}
+          onMenuItemClick={onMenuItemClick && (() => onMenuItemClick(id))}
+          onGroupItemClick={onGroupItemClick && (groupMenuId => onGroupItemClick(id, groupMenuId))}
+        />
+      )
+    })}
+  </Flex>
+)
+
+export const MenuSidebar = styled(Flex).attrs({
+  overflow: { vertical: "auto" },
+})`
+  position: sticky;
```

Comment:
we need to fix those styles. With a flex, flex: 1 and height of 100% on its parent it should work as needed.

Reply:
This style and some others will be fixed once i refactor the charts view. Let's leave it as it was for now.